sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/schema.yaml

```diff
@@ -0,0 +1,19 @@
+fields:
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+- mode: NULLABLE
+  name: version_major
+  type: NUMERIC
+- mode: NULLABLE
+  name: geo_country
+  type: STRING
+- mode: NULLABLE
+  name: platform
+  type: STRING
+- mode: NULLABLE
+  name: metric
+  type: STRING
+- mode: NULLABLE
+  name: rate
```

Comment:
Would be great to have descriptions of the fields, at least for the less familiar ones. E.g. what this rate is counting.

Reply:
Added them in, thank you for the suggestion!